src/worker-error-reporting.js

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
Can we move this into a common directory? Ie, `/worker`, with `/worker/service-worker` and `/worker/web-worker`?